tests/chart_tests/test_authsidecar.py

```diff
@@ -126,7 +126,15 @@ def test_authSidecar_houston_configmap_without_annotation(self, kube_version):
         """Test Houston Configmap with authSidecar."""
         docs = render_chart(
             kube_version=kube_version,
-            values={"global": {"authSidecar": {"enabled": True}}},
+            values={
+                "global": {
+                    "authSidecar": {
+                        "enabled": True,
+                        "repository": "quay.io/astronomer/ap-auth-sidecar",
```

Comment:
Can we make this something that is nothing like what we would actually use so when it fails it's clear that the failure isn't related to the defaults?